app/scripts/proactive/rest/studio-client.js

```diff
@@ -230,6 +230,27 @@ define(
                 });
             },
 
+            getDomains : function () {
+                var domains=[];
+                $.ajax({
+                    type: 'GET',
+                    url: config.restApiUrl + "/domains",
+                    async: false,
+                    success: function(data) {
+                        if (data.length > 0 && !data.includes("")) {
+                            domains.push("");
```

Comment:
Why do you add an empty string at the end of the list if there is returned data ?

Reply:
I add an empty string at the begining of the list so that the user has the possibility to login using a domain or not

Reply:
Ok thanks, that makes sense.